code/FBX/FBXConverter.cpp

```diff
@@ -120,6 +125,46 @@ namespace Assimp {
             ConvertGlobalSettings();
             TransferDataToScene();
 
+            // Now convert all bone positions to the correct mOffsetMatrix
```

Comment:
Bone stack handling

A bone stack is simply as it sounds, you can have many armatures and many bones with the same names with this method, it will always provided 1:1 from a package to an engine, and allow for string based matches with duplicates.

It's a pretty neat way of dealing with it as later it lowers the amount of code you need in your importer and simplifies down so you don't have to make bones unique and then losing their armature aiNode counterpart link in the process. :)